### PR TITLE
Update tic_tac_toe.py

### PR DESCRIPTION
**Pull request by harshitashyale**
Minor code change

### CODE REVIEW

File: tic_tac_toe.py [hunk 1]
```diff
@@ -2,7 +2,7 @@ def sum(a, b, c ):
     return a + b + c
 
 
-def printBoard(playerX, playerY):
+def printBoard(playerY, playerX):
```
Comment (harshitashyale) [inline tic_tac_toe.py:5]: Reviewer Comments

**Comment by harshitashyale** [comment]
hey this is stupid

File: tic_tac_toe.py
Comment (harshitashyale) [inline tic_tac_toe.py:1]: general comment

Comment (harshitashyale) [inline tic_tac_toe.py:1]: hello this is to start a review

File: tic_tac_toe.py [hunk 2]
```diff
@@ -2,7 +2,7 @@ def sum(a, b, c ):
     return a + b + c
 
 
-def printBoard(playerX, playerY):
+def printBoard(playerY, playerX):
     zero = 'X' if playerX[0] else ('O' if playerY[0] else 0)
     one = 'X' if playerX[1] else ('O' if playerY[1] else 1)
     two = 'X' if playerX[2] else ('O' if playerY[2] else 2)
```
Comment (harshitashyale) [inline tic_tac_toe.py:8]: hello this is second review comment

File: tic_tac_toe.py (hunk 1 above)
Comment (jaredsb) [inline tic_tac_toe.py:5]:
```suggestion
def printBoard2(playerY, playerX):
```

**Review by jaredsb — CHANGES_REQUESTED** [review]
Make sure to make this update!

**Pull request closed without merging**